src/modules/finance/index.ts

```diff
@@ -486,23 +491,56 @@ export class FinanceModule extends ModuleBase {
   /**
    * Generates a random Bitcoin address.
    *
+   * @param options An options object.
+   * @param options.type The bitcoin address type (`'legacy'`, `'sewgit'`, `'bech32'` or `'taproot'`). Defaults to `'legacy'`.
+   * @param options.network The bitcoin network (`'mainnet'` or `'testnet'`). Defaults to `'mainnet'`.
+   *
    * @example
-   * faker.finance.bitcoinAddress() // '3ySdvCkTLVy7gKD4j6JfSaf5d'
+   * faker.finance.bitcoinAddress() // '1TeZEFLmGPLEQrSRdAcnZLoWwYeiHwmRog'
+   *
+   * enum BitcoinAddressType { Legacy, Segwit, Bech32, Taproot }
+   * faker.finance.bitcoinAddress({ type: BitcoinAddressType.Bech32 }) // 'bc1qw508d6qejxtdg4y5r3zarvary0c5xw7kv8f3t4'
+   *
+   * enum BitcoinAddressType { Legacy, Segwit, Bech32, Taproot }
+   * enum BitcoinNetwork { Mainnet, Testnet }
+   * faker.finance.bitcoinAddress({ type: BitcoinAddressType.Bech32, network: BitcoinNetwork.Testnet }) // 'tb1qw508d6qejxtdg4y5r3zarvary0c5xw7kxpjzsx'
```

Comment:
I don't think that it's required to include the enums in the `@example` section. Maybe add `@see` to point to them? Simply lsiting examples including the values should eb sufficent.

```suggestion
   * faker.finance.bitcoinAddress() // '1TeZEFLmGPLEQrSRdAcnZLoWwYeiHwmRog'
   * faker.finance.bitcoinAddress({ type: BitcoinAddressType.Bech32 }) // 'bc1qw508d6qejxtdg4y5r3zarvary0c5xw7kv8f3t4'
   * faker.finance.bitcoinAddress({ type: BitcoinAddressType.Bech32, network: BitcoinNetwork.Testnet }) // 'tb1qw508d6qejxtdg4y5r3zarvary0c5xw7kxpjzsx'
```

Reply:
`npm run preflight` fails without them:
```
 FAIL  test/scripts/apidocs/verify-jsdoc-tags.spec.ts > verify JSDoc tags > finance > bitcoinAddress > 0 > verify @deprecated tag
ReferenceError: BitcoinAddressType is not defined
 ❯ test/scripts/apidocs/temp/finance/bitcoinAddress_0.ts:4:60
 ❯ test/scripts/apidocs/verify-jsdoc-tags.spec.ts:185:17
    183| 
    184|                 // Run the examples
    185|                 await import(`${path}?scope=deprecated`);
       |                 ^
    186| 
    187|                 // Verify that deprecated methods log a warning
```

The examples for `enumValue` under `src/modules/helpers/index.ts` use a similar approach. Is there any way around it ?

Reply:
You have to export the enums in the main index.ts.
If that doesnt help, I have to check our import detection code.

Reply:
Exporting in main `index.ts` didn't work for the raw enums.However I've noticed that other enums are following a different pattern of exporting an additional `<enum>Type` with the enum values as a string, like `Sex` and `SexType` for example. I've changed `BitcoinAddressType` and `BitcoinNetwork` to follow this pattern, and removed the imports from the `@example` tag.

Reply:
https://github.com/faker-js/faker/pull/2922/commits/4f6f03b481537b8f47f0e2c7ba322944c981d3dc